corehq/apps/data_dictionary/static/data_dictionary/js/data_dictionary.js

```diff
@@ -235,7 +235,7 @@ hqDefine("data_dictionary/js/data_dictionary", [
                             'caseType': element.caseType,
                             'name': element.name,
                             'label': element.label() || element.name,
-                            'index': index,
+                            'index': group.toBeDeprecated() ? 0 : index,
```

Comment:
If we deprecate the group and then undeprecate it again before saving, would all the case properties be saved with an index of 0?

Reply:
No, this is only when saved is called.

Reply:
Thanks for clarifying, that makes sense.